api/services/handle_event_service.py

```diff
@@ -61,5 +71,5 @@ def handle_event_service(self, form_data):
                 registration.RegistrationService.create_registration(self, data)
 
         except Exception as e:
-            print(e)
-            print("Failed to log the call details")
+            logger.error("Failed to log the call details: %s", str(e))

```

Comment:
Add user detail like phone number in the message

Reply:
Updated